templates/donor-donation-refund-tip.twig

```diff
@@ -0,0 +1,16 @@
+{% extends renderHtml ? '_base.html.twig' : '_base.text.twig' %}
+
+{% block content %}
```

Comment:
Is it worth considering a mention of the charity's name somewhere in this email, to help donors recall what it relates to?